tests/console/orphaned_packages_check.pm

```diff
@@ -59,7 +60,12 @@ sub run {
 
     record_info((is_offline_upgrade_or_livecd) ? 'Upgrade/LiveCD' : 'No upgrade/LiveCD', 'Upgraded or installed from LIVECD can possibly cause orphans');
 
-    # Orphans are also expected in JeOS without SDK module (jeos-firstboot, jeos-license and live-langset-data)
+    if (is_sle() && is_jeos()) {
```

Comment:
My fault that I haven't mentioned this earlier! :facepalm: 

You would have achieved the same with `SCC_ADDONS=base,serverapp,desktop,sdk` as you need to use `console/suseconnect_scc.pm` to register the system. Sorry!